packages/ember-metal/lib/deprecate_property.js

```diff
@@ -23,7 +23,7 @@ import { set } from "ember-metal/property_set";
 
 export function deprecateProperty(object, deprecatedKey, newKey) {
   function deprecate() {
-    Ember.deprecate('Usage of `' + deprecatedKey + '` is deprecated, use `' + newKey + '` instead.');
+    Ember.deprecate(`Usage of "${deprecatedKey}" is deprecated, use "${newKey}" instead.`);
```

Comment:
Agreed, it lets the developer know that it's an exact attribute and that's what they should look for to fix this.